filament/backend/include/backend/DriverEnums.h

```diff
@@ -977,31 +977,29 @@ struct StencilState {
 
         uint8_t padding1                                : 2;                    // 16
 
+        //! Reference value for stencil comparison tests and updates
+        uint8_t ref = 0;                                                        // 24
+
         //! Masks the bits of the stencil values participating in the stencil comparison test.
-        uint8_t readMask;                                                       // 24
+        uint8_t readMask = 0xff;                                                // 32
 
         //! Masks the bits of the stencil values updated by the stencil test.
-        uint8_t writeMask;                                                      // 32
+        uint8_t writeMask = 0xff;                                               // 40
     };
 
     //! Stencil operations for front-facing polygons
-    StencilOperations front = {
-            .stencilFunc = StencilFunction::A, .readMask = 0xff, .writeMask = 0xff };
+    StencilOperations front = { .stencilFunc = StencilFunction::A };
```

Comment:
Could we put the `stencilFunc` initializer as part of the `StencilOperations` class and leave this as `= {}`?

Reply:
I don't think we can because it's a bitfield. Or we have to create a ctor, but then we can't use designated initializers.

Maybe it would be more logical to not do any default initialization at all in `StencilOperations` and leave it to `StencilStace`.